tools/install_visual_studio_22.ps1

```diff
@@ -0,0 +1,12 @@
+# Helper script to automate installing Visual Studio 22 and required components.
+
+Invoke-WebRequest -Uri https://aka.ms/vs/17/release/vs_professional.exe `
+    -OutFile C:\src\vs_professional.exe
```

Comment:
Left ( late) comments for this on go/cobalt-cl/246586, but could we try to use a TMP or TEMP environment variable here instead of `C:\src` ? The directory doesn't necessarily exist for everyone.